modules/ding_sections/ding_sections.admin.inc

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @file
+ * Ding section dashboard administrative interface.
+ */
+
+/**
+ * Creates an administrative block on the configuration page.
+ *
+ * It is necessary for grouping various settings.
+
+ * @return string
+ *   HTML for the block.
+ */
+function ding_sections_admin_menu_block_page() {
+  $item = menu_get_item();
+  $content = system_admin_menu_block($item);
+
+  if ($content) {
+    $output = theme('admin_block_content', array('content' => $content));
+  }
+  else {
+    $output = t('You do not have any administrative items.');
+  }
+
+  return $output;
+}
+
+/**
+ * Content type selection form.
+ *
+ * Interface for selecting node types which should be a part
+ * of teh section system.
+ */
+function ding_sections_admin_content_types($form, $form_state) {
+  $node_types = node_type_get_types();
+  $node_bundles = array();
+
+  foreach ($node_types as $type) {
+    $node_bundles[$type->type] = $type->name;
+  }
+
+  $form['section_content_types'] = array(
+    '#type' => 'checkboxes',
+    '#title' => t('Available node types'),
+    '#description' => t('Select which nodes should be a part of section system'),
+    '#options' => $node_bundles,
+    '#default_value' => variable_get('section_content_types', array()),
```

Comment:
The variable (and index in $form) should be prefixed with ding_ (so it's ding_section_content_types).